acceptance/acceptance_test.go

```diff
@@ -0,0 +1,88 @@
+// +build acceptance
+
+package acceptance
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	h "github.com/buildpack/lifecycle/testhelpers"
+)
+
+func TestAcceptance(t *testing.T) {
+	spec.Run(t, "acceptance", testAcceptance, spec.Sequential(), spec.Report(report.Terminal{}))
+}
+
+func testAcceptance(t *testing.T, when spec.G, it spec.S) {
+
+	var lifecycleCmd = func(name string, args ...string) *exec.Cmd {
```

Comment:
Separate into real function with `t` as first arg.

Also, for all testing helpers, call `t.Helper()`.